server/test/conversations.spec.js

```diff
@@ -0,0 +1,76 @@
+require('dotenv').config();
+const chai = require('chai');
+const chaiHttp = require('chai-http');
+const app = require('../app.js');
+const { createUsers } = require('../utils/testUtils');
+
+const should = chai.should();
+chai.use(chaiHttp);
+
+let users = [];
+let singleConversationId = '';
+
+before('Create Users', (done) => {
+  users = createUsers(2);
+  done();
+});
+
+const checkSingleConversation = (err, res) => {
+  should.not.exist(err);
+  res.should.have.status(200);
+
+  res.body.should.contain.all.keys([
+    'users',
+    'messages',
+    'created'
+  ]);
+
+  res.body.users.should.be.an('array').that.has.lengthOf(2);
+};
+
+describe('Conversations', () => {
```

Comment:
There should be tests that fail to create conversations as well. Same with message tests. The passing tests can only be trusted if you can prove that failure exists.